src/tic_tac_toe_pair/routes.clj

```diff
@@ -0,0 +1,19 @@
+(ns tic-tac-toe-pair.routes
+  (:require [compojure.core :refer :all]
+            [compojure.route :as route]
+            [ring.middleware.json :as ring-json]
+            [tic-tac-toe-pair.handlers :refer :all]))
+
+(def json-routes
+    (routes
+        (POST "/player" [] (ring-json/wrap-json-body player-handler))
+        (POST "/move" [] (ring-json/wrap-json-body move-handler))))
+
+    
+(defroutes non-body-routes 
```

Comment:
Are these non-body-routes only needed for testing? I see one test where you're hitting the `/trouble` endpoint to verify your error handler is working as expected. Could you mock one of your handlers for a real route to force it to throw an exception instead?